src/components/CopyWithText.tsx

```diff
@@ -28,7 +28,7 @@ export const CopyWithText = ({
     <>
       {children}
       <CopyToClipboard text={textToCopy} onCopy={handleCopy}>
-        <TextButton>{inProgress ? "Copied" : "Copy"}</TextButton>
+        <TextLink role="button">{inProgress ? "Copied" : "Copy"}</TextLink>
```

Comment:
Instead of using a string for the role in `role="button"` we could use an enum. This is probably something for SDS though.

Reply:
🤔 Hmm, I'm not sure we need to create an enum for roles, because the values are coming from [HTML](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques).